app/Nova/FiscalYear.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova;
+
+use App\Models\FiscalYear as AppModelsFiscalYear;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\HasMany;
+use Laravel\Nova\Fields\Number;
+
+class FiscalYear extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = AppModelsFiscalYear::class;
+
+    /**
+     * The single value that should be used to represent the resource when being displayed.
+     *
+     * @var string
+     */
+    public static $title = 'ending_year';
+
+    /**
+     * The columns that should be searched.
+     *
+     * @var array<string>
+     */
+    public static $search = [
+        'ending_year',
+    ];
+
+    /**
+     * Get the fields displayed by the resource.
+     *
+     * @return array<\Laravel\Nova\Fields\Field>
+     */
+    public function fields(Request $request): array
+    {
+        return [
+            Number::make('Ending Year')
+                ->rules('required', 'max:255')
```

Comment:
This should ensure the year is >=2000 or similar so people don't enter two digit numbers.